passenger_car_kinematic_model/include/passenger_car_kinematic_model/PassengerCarKinematicModel.h

```diff
@@ -47,9 +54,19 @@ class PassengerCarKinematicModel: public lib_vehicle_model::VehicleMotionModel
     std::shared_ptr<lib_vehicle_model::ParameterServer> param_server_;
     
     // Parameters
-    double l_; // Wheel base in m
-    double R_ef_; // The vertical distance from the front axle to the ground when the vehicle is loaded. 
-    double R_er_; // The vertical distance from the rear axle to the ground when the vehicle is loaded. 
+    double l_f_; // Distance from front wheel to CG in m
```

Comment:
Do you think those variable should be initialized? I think they will have non-deterministic value if getParam() fails.